compiler/damlc/tests/src/daml-ghc-deterministic.sh

```diff
@@ -28,6 +28,7 @@ TESTS_DIR=$(dirname $(rlocation "$TEST_WORKSPACE/compiler/damlc/tests/daml-test-
 damlc=$(rlocation "$TEST_WORKSPACE/$1")
 protoc=$(rlocation "$TEST_WORKSPACE/$2")
 diff="$3"
+scenarios="--enable-scenarios=yes" # TODO: https://github.com/digital-asset/daml/issues/11316
```

Comment:
not sure why there is even a scenario in here, we should just use a module without one.

Reply:
ah that's fair, do you have one in mind?

Reply:
could take the `User.daml` file from create-daml-app for example.